src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/ResidentialChecksService.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.model.ResidentialChecksTaskProgress
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.model.ResidentialChecksView
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.policy.RESIDENTIAL_CHECKS_POLICY_V1
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.policy.model.residentialchecks.ResidentialChecksStatus
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service.policy.model.residentialchecks.ResidentialChecksTaskStatus
+
+@Service
+class ResidentialChecksService(
+  private val assessmentService: AssessmentService,
+) {
+  fun getResidentialChecksView(prisonNumber: String): ResidentialChecksView {
+    val currentAssessment = assessmentService.getCurrentAssessmentSummary(prisonNumber)
+
+    val tasks = RESIDENTIAL_CHECKS_POLICY_V1.tasks.map { task ->
+      ResidentialChecksTaskProgress(
+        code = task.code,
+        taskName = task.name,
+        status = ResidentialChecksTaskStatus.NOT_STARTED,
```

Comment:
All status will be NOT_STARTED for now as we have no way of persisting / updating status of tasks yet.